lmdk/include/sink_api_implementation.h

```diff
@@ -73,6 +73,7 @@ struct sof_sink {
 	const struct sink_ops *ops;	/** operations interface */
 	size_t requested_write_frag_size; /** keeps number of bytes requested by get_buffer() */
 	size_t num_of_bytes_processed; /** processed bytes counter */
+	size_t obs;
```

Comment:
better use last implementation - IBS/OBS has been replaced by more meaningful names min_free_space / min_available

Reply:
We should probably also string change that change in code, manifest and docs too.

Reply:
this change is based on MTL-006 branch, so it is copier from base fw part.  When Guennadi come back, he will do it for main branch.